docs/data.rst

```diff
@@ -641,6 +641,51 @@ then when you run the command above it will run the recipe
 100 times (100*10=1000) which will generate 1000 Accounts, 500 Contacts
 and 1500 Opportunities.
 
+Controlling the Loading Process
+-------------------------------
+
+CumulusCI's data loader has many knobs and switches that you might want to
+adjust during your load. It supports a ".load.yml" file format which allows you to
+manipulate these load settings. The simplest way to use this file format is to make
+a file in the same directory as your recipe with a filename that is derived from
+the recipe's by replacing everything after the first "." with ".load.yml". For example,
+if your recipe is called "babka.recipe.yml" then your load file would be
+"babka.load.yml".
+
+Inside of that file you put a list of declarations in the following format:
+
+```yaml
+- sf_object: Account
+  api: bulk
+  bulk_mode: parallel
+```
+
+Which would specifically load accounts using the bulk API's parallel mode.
+
+The specific keys that you can associate with an object are:
+
+* api: "smart", "rest" or "bulk"
+* batch_size: a number
+* bulk_mode: "serial" or "parallel"]
+* load_after: the name of another sobject to wait for before loading
```

Comment:
These docs would be more useful if you explained what the options do / when to use them -- but I haven't checked to see what is covered already in the context of mapping files.

Reply:
Link added.